.github/integ-config/integ-datastore.yml

```diff
@@ -1,120 +1,134 @@
+#   spec: multi-auth-three-plus-rules
+#   browser: minimal_browser_list
 
-# integ_react_datastore_subs_disabled_v2
-- test_name: 'DataStore - Subs Disabled CLI v2'
-  framework: react
-  category: datastore
-  sample_name: v2/subs-disabled-v2
-  spec: subs-disabled
-  browser: minimal_browser_list
+# # integ_react_datastore_subs_disabled
+# - test_name: 'DataStore - Subs Disabled'
+#   framework: react
+#   category: datastore
+#   sample_name: subs-disabled
+#   spec: subs-disabled
+#   browser: minimal_browser_list
 
-# integ_react_datastore_consecutive_saves
-- test_name: 'DataStore - Subs Disabled'
-  framework: react
-  category: datastore
-  sample_name: consecutive-saves
-  spec: consecutive-saves
-  browser: minimal_browser_list
+# # integ_react_datastore_subs_disabled_v2
+# - test_name: 'DataStore - Subs Disabled CLI v2'
+#   framework: react
+#   category: datastore
+#   sample_name: v2/subs-disabled-v2
+#   spec: subs-disabled
+#   browser: minimal_browser_list
 
-# integ_react_datastore_consecutive_saves_v2
-- test_name: 'DataStore - Subs Disabled CLI v2'
-  framework: react
-  category: datastore
-  sample_name: v2/consecutive-saves-v2
-  spec: consecutive-saves
-  browser: minimal_browser_list
+# # integ_react_datastore_consecutive_saves
+# - test_name: 'DataStore - Subs Disabled'
+#   framework: react
+#   category: datastore
+#   sample_name: consecutive-saves
+#   spec: consecutive-saves
+#   browser: minimal_browser_list
 
-# # integ_react_datastore_observe_query
-# - test_name: 'DataStore - Observe Query'
+# # integ_react_datastore_consecutive_saves_v2
+# - test_name: 'DataStore - Subs Disabled CLI v2'
 #   framework: react
 #   category: datastore
-#   sample_name: observe-query
-#   spec: observe-query
+#   sample_name: v2/consecutive-saves-v2
+#   spec: consecutive-saves
 #   browser: minimal_browser_list
 
-# # integ_react_datastore_observe_query_v2
-# - test_name: 'DataStore - Observe Query CLI v2'
+# # # integ_react_datastore_observe_query
+# # - test_name: 'DataStore - Observe Query'
+# #   framework: react
+# #   category: datastore
+# #   sample_name: observe-query
+# #   spec: observe-query
+# #   browser: minimal_browser_list
+
+# # # integ_react_datastore_observe_query_v2
+# # - test_name: 'DataStore - Observe Query CLI v2'
+# #   framework: react
+# #   category: datastore
+# #   sample_name: v2/observe-query-v2
+# #   spec: observe-query
+# #   browser: minimal_browser_list
+
+# # integ_react_datastore_schema_drift
+# - test_name: 'DataStore - Schema Drift'
 #   framework: react
 #   category: datastore
-#   sample_name: v2/observe-query-v2
-#   spec: observe-query
+#   sample_name: schema-drift
+#   spec: schema-drift
 #   browser: minimal_browser_list
 
-# integ_react_datastore_schema_drift
-- test_name: 'DataStore - Schema Drift'
-  framework: react
-  category: datastore
-  sample_name: schema-drift
-  spec: schema-drift
-  browser: minimal_browser_list
+# integ_datastore_auth
```

Comment:
Only two new tests are added.
Apart from this no existing test has been modified (just commented)